libs/wingc/src/jsify.rs

```diff
@@ -138,6 +142,7 @@ impl<'a> JSifier<'a> {
 		let mut output = CodeMaker::default();
 
 		let is_entrypoint_file = source_path == self.entrypoint_file_path;
+		let is_directory = source_path.is_dir();
```

Comment:
should we validate that this is a relative path (without `..`), otherwise the graph won't include it, right?

Reply:
Mmm -- I've gone ahead and added some validation to `parser.rs` for validating we're dealing with relative paths (it kind of feels like this kind of input validation should be handled long before jsification)